metabase/metabase
# Add unsubscribe user modal in admin

A premium feature to unsubscribe a user from all dashboard subscriptions and alerts and delete ones the user has created. 

How to test:
- Run `yarn dev-ee`
- Create a question with an alert
- Navigate to `Admin > People`
- In the context menu for a user select `Unsubscribe from all subscriptions / alerts`
- `Confirm` unsubscribing
- Check that the alert was archived, e.g. by navigating to the question or to `Account settings > Notifications`

<img width="630" alt="Screen Shot 2021-09-15 at 1 54 08 pm" src="https://user-images.githubusercontent.com/8542534/133421724-5d75607b-2f36-46bc-bddb-2b96cae58365.png">
<img width="451" alt="Screen Shot 2021-09-15 at 1 59 23 pm" src="https://user-images.githubusercontent.com/8542534/133421727-79843f78-72df-4c56-b27d-9bd9b9edcb7d.png">

Please note that the corresponding cypress test fails as the BE endpoint does not delete slack subscriptions; to be fixed separately.
